docs/tutorial/part-zero/index.md

```diff
@@ -66,8 +66,14 @@ The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run com
 
 The Gatsby CLI is available via npm and should be installed globally by running `npm install -g gatsby-cli`.
 
+To see the latest gatsby-cli version installed run `gatsby --version`.
+
+![Check gatsby version in terminal](05-gatsby-latest-version.png)
```

Comment:
Not sure if these are necessary in the tutorial. It's difficult to find a balance too little information and too much. 

@marcysutton What do you think?

Reply:
I don't think checking the version is necessary, come to think of it. The CLI prompts users to install new versions anyway!

Reply:
Yeah agreed.

@sumitkharche could you remove all the changes except the help command? We don't add periods to the end of sentences in bullets on gatsbyjs.org so the periods were deliberately not there.